src/markdown-pages/install-with-kurl/managing-nodes.md

```diff
@@ -0,0 +1,247 @@
+
+For information about how to remove a node from a cluster that does not use Rook Ceph, see [kubectl drain](https://kubernetes.io/docs/tasks/administer-cluster/safely-drain-node/) in the Kubernetes documentation.
+
+### Rook Ceph and etcd Node Removal Requirements
+
+Review the following requirements and considerations before you remove one or more nodes from Rook Ceph and etcd clusters:
+
+* **etcd cluster health**: To remove a primary node from etcd clusters, you must meet the following requirements to maintain etcd quorum:
+   * You must have at least one primary node.
+   * If you scale the etcd cluster to three primary nodes, you must then maintain a minimum of three primary nodes to maintain quorum.
+* **Rook Ceph cluster health**: When you scale a Ceph Storage Cluster to three or more Ceph Object Storage Daemons (OSDs), such as when you add additional manager or worker nodes to the cluster, the Ceph Storage Cluster can no longer have fewer than three OSDs. If you reduce the number of OSDs to less than three in this case, then the Ceph Storage Cluster loses quorum.
+* **Add a node before removing a node**: To remove and replace a node, it is recommended that you add a new node first before removing the node. For example, to remove one node in a three-node cluster, first add a new node to scale the cluster to four nodes. Then, remove the desired node to scale the cluster back down to three nodes.
+* **Remove one node at a time**: If you need to remove multiple nodes from a cluster, remove one node at a time.
+
+### Rook Ceph Cluster Prerequisites
+
+Complete the following prerequisites before you remove one or more nodes from a Rook Ceph cluster:
+
+* (Recommended) Upgrade Rook Ceph to v1.4 or later.
+
+* In the kURL specification, set `isBlockStorageEnabled` to `true`. This is the default for Rook Ceph v1.4 and later.
+
+* Ensure that you can access the ceph CLI from a Pod that can communicate with the Ceph Storage Cluster. To access the ceph CLI, you can do one of the following:      
+
+  * (Recommended) Use the `rook-ceph-tools` Pod to access the ceph CLI. Use the same version of the Rook toolbox as the version of Rook Ceph that is installed in the cluster.
+
+     For information about how to create a `rook-ceph-tools` Pod for Rook Ceph v1.5, see [Rook Toolbox](https://rook.io/docs/rook/v1.5/ceph-toolbox.html) in the Rook Ceph documentation.
+
+  * Use `kubectl exec` to enter the `rook-ceph-operator` Pod, where the ceph CLI is available.
+
+     You can either open an interactive shell to run multiple ceph CLI commands in a row, or you can precede each ceph CLI command that you run with `kubectl exec`. The following shows an example of each method:
+
+ * (Optional) Open an interactive shell in the `rook-ceph-tools` or `rook-ceph-operator` Pod to run multiple ceph CLI commands in a row.
```

Comment:
It's hard to tell without a preview, but I think it might be easier to read if you move this as an intro after "Example:"

And start this option with "**Example: Using an Interactive Shell** 

And then do something similar with the other method (non-interactive shell).

Reply:
Ah good call, yeah, this section is unnecessarily complex in terms of these little headings. Reworking